detekt.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analysers
 description=Provide detekt rules for Kotlin projects
 homepageUrl=https://github.com/detekt/sonar-kotlin
-archivedVersions=
-publicVersions=2.0.0
+archivedVersions=2.0.0
+publicVersions=2.1.0
 
 defaults.mavenGroupId=io.github.detekt
 defaults.mavenArtifactId=sonar-detekt
 
+2.1.0.description=Upgrade to detekt 1.9.1
+2.1.0.sqVersions=[7.9,LATEST]
+2.1.0.date=2020-06-02
+2.1.0.changelogUrl=https://github.com/detekt/sonar-kotlin/releases/tag/2.1.0
+2.1.0.downloadUrl=https://github.com/detekt/sonar-kotlin/releases/download/2.1.0/sonar-detekt-2.1.0.jar
+
 2.0.0.description=Initial Marketplace entry
-2.0.0.sqVersions=[7.9,LATEST]
+2.0.0.sqVersions=[7.9,8.2]
```

Comment:
Unless you know of a specific incompatibility with 8.3, you want to set the end range to that. Why? Because users might be on 8.3 and this version of your plugin. If the compatibility range doesn't include 8.3 then those users won't be prompted to upgrade in the Marketplace.

Reply:
Is it recommend to still write `LATEST` here or should I update the numbers to the latest released sonarqube version after I've verified it still works with it?